styles/migadu-webmail/catppuccin.user.css

```diff
@@ -0,0 +1,421 @@
+    --dropdown-menu-disable-color: @overlay0;
+    --dropdown-menu-border-clr: @surface0;
+    --folders-color: @text;
+    --folders-disabled-color: @surface2;
+    --folders-selected-color: @blue;
+    --folders-selected-bg-color: fade(@blue, 10%);
+    --folders-focused-color: @blue;
+    --folders-hover-color: @text;
+    --folders-hover-bg-color: @base;
+    --folders-drop-color: @text;
+    --settings-menu-color: @text;
+    --settings-menu-selected-color: @blue;
+    --settings-menu-selected-bg-color: fade(@blue, 10%);
+    --settings-menu-hover-color: @text;
+    --settings-menu-hover-bg-color: @base;
+    --message-list-toolbar-bg-color: @overlay0;
+    --panel-bg-clr: @base;
+    --message-header-bg-clr: @mantle;
+    --hr-color: @surface0;
+    --dialog-bg-clr: @base;
+    --dialog-clr: @text;
+    --tr-hover-bg-clr: @crust;
+    --loading-color: @text;
+    --login-color: @text;
+    --login-bg-color: @crust;
+      --tab-hover-border-clr: @surface0;
+      --tab-active-bg-clr: @crust;
+      --tr-odd-bg-clr: @mantle;
+    .LoginView .btn,
+    .LoginView input {
+      border-color: @surface0;
+    }
+    .LoginView input:focus,
+    .LoginView input:hover {
+      border-color: @surface1;
+    }
+    .LoginView input::placeholder {
+      color: var(--login-color);
+    }
+
+    .messageItemHeader,
+    .bodySubHeader,
+    .thm-message-list-top-toolbar,
+    .thm-message-list-bottom-toolbar {
+      background: @mantle !important;
+    }
+
+    /* Buttons */
+
+    --btn-clr: @text;
+    --btn-border-clr: @overlay0;
+    .btn.disabled {
+      color: @surface0;
+    }
+    .btn.btn-success:hover,
+    .btn.btn-success:active {
+      background-color: desaturate(lighten(fade(@green, 40%), 10%), 10%);
+    }
+    .btn.btn-success,
+    .btn.btn-success [class*=" icon-"],
+    .btn.btn-success [class^="icon-"] {
+      color: @green;
+    }
+    .btn.disabled.fontastic,
+    .btn.disabled .fontastic {
+      color: @surface0;
+    }
+    .btn,
+    .btn.disabled,
+    .btn[disabled],
+    .btn.disabled:hover,
+    .btn[disabled]:hover {
+      border-color: @surface0 !important;
+      text-shadow: none !important;
+      background-color: @base;
+    }
+
+    /* Links */
+
+    --link-color: @blue;
+    #messageItem .b-text-part a:visited {
+      color: @mauve;
+    }
+
+    /* Inputs */
+
+    --input-bg-clr: @base;
+    --input-border-clr: @surface0;
+    --input-focus-border-clr: @overlay0;
+    --input-clr: @subtext1;
+
+    input::placeholder {
+      color: @overlay2;
+    }
+
+    .select,
+    select {
+      @svg: escape(
+        '<svg xmlns="http://www.w3.org/2000/svg" height="10px" width="15px"><text x="0" y="10" fill="@{subtext1}">▾</text></svg>'
+      );
+      background-image: url("data:image/svg+xml,@{svg}") !important;
+    }
+
+    .e-checkbox.material-design input:checked + div {
+      border-color: darken(@green, 10%);
+    }
+
+    /* stylelint-disable property-disallowed-list */
```

Comment:
I have to use `border` here (and all the other non-color properties) because I have to completely redeclare each keyframe -_-.